pkg/cloud/cloud_test.go

```diff
@@ -82,7 +82,51 @@ func extractVolumeIdentifiers(volumes []types.Volume) (volumeIDs []string, volum
 	}
 	return volumeIDs, volumeNames
 }
+func TestNewCloud(t *testing.T) {
 
+	testCases := []struct {
+		name            string
+		region          string
+		awsSdkDebugLog  bool
+		userAgentExtra  string
+		batchingEnabled bool
+	}{
+		{
+			name:            "success: with debug, userAgentExtra, and batching",
```

Comment:
np: Specify the full parameter names as done for `userAgentExtra`
- `debug` -> `awsSdkDebugLog`
- `batching` -> `batchingEnabled`